terraform/gcp/modules/monitoring/dex/variables.tf

```diff
@@ -44,10 +44,10 @@ variable "dex_url" {
 
 // Set-up for notification channel for alerting
 variable "notification_channel_id" {
```

Comment:
not sure if it is worthwhile to do or not, but rename to "notification_channel_ids"

and updating the description to indicate it takes more than one id?

same in other modules

Reply:
done